Symfony/CS/Tests/ConfigurationResolverTest.php

```diff
@@ -69,7 +68,8 @@ public function testResolveFixersReturnsEmptyArrayByDefault()
 
     public function testResolveFixersWithLevelConfig()
     {
-        $this->config->level(FixerInterface::PSR1_LEVEL);
+        $this->config->expects($this->once())->method('getLevel')
```

Comment:
You don't know how many times the `getLevel` will be called, and the getter returns const value (the value shouldn't changed between calling).
I would suggest `->any()` here and in other places